### DISCUSSION TITLE
Mp3 media files stop midway

### DISCUSSION THREAD

**Discussion by scouech** [directus/directus#5320] [Archive]
Hi everyone!

I'm using the latest version of directus with nginx and postgre, host on digitalocean.

We have a lot of audio files (mp3) that people can listen to but it seems that for long files (15 minutes and above) the connection stops while people are listening.

I know it's 99% sure it's not because of directus, but I'm wondering if you have an idea on how to fix this? Probably on nginx side?

Cheers

**Comment by rijkvanzanten** [comment]
Very strange! Do you have an example of a file where this happens? Could have something to do with the Range header (for streaming) returning the wrong data? Though that would be a bit strange for it failing on the 15 minutes mark exactly instead of a certain number of bytes transferred

**Reply by scouech** [reply]
mmm I don't have an example via the browser. We are using directus for our app and a few beta testers have reported this problem. I did check and it's not really 15 minutes but more while listening to an mp3 that is at least 10 minutes long.. doesn't seem to happen for shorter files...
I will mark your comment as answer for now because I might be waisting your time as we need to be sure it doesn't come from our end! Sorry about that! Will let you know if we clear out our side :)

**Reply by rijkvanzanten** [reply]
Oki! I'll gladly take a look if we can reproduce the problem in Directus itself 🙂
